src/testing/h_refinement_study_isentropic_vortex.cpp

```diff
@@ -127,9 +127,40 @@ int HRefinementStudyIsentropicVortex<dim, nstate>::run_test() const
     }
 
     int testfail = 0;
+// cESFR range test -------------------------------------
+   
+    const unsigned int nb_c_value = this->all_parameters->number_ESFR_parameter_values;
+    const double c_min = this->all_parameters->ESFR_parameter_values_start;
+    const double c_max = this->all_parameters->ESFR_parameter_values_end;
+    const double log_c_min = std::log10(c_min);
+    const double log_c_max = std::log10(c_max);
+    std::vector<double> c_array(nb_c_value+1);
+
+
+    // Create log space array of c_value
+    for (unsigned int ic = 0; ic < nb_c_value; ic++) {
+        double log_c = log_c_min + (log_c_max - log_c_min) / (nb_c_value - 1) * ic;
+        c_array[ic] = std::pow(10.0, log_c);
+    }
+    if (this->all_parameters->flux_reconstruction_type == PHiLiP::Parameters::AllParameters::Flux_Reconstruction::user_specified_value){
+        // Add cPlus value at the end
+        c_array[nb_c_value] = this->all_parameters->FR_user_specified_correction_parameter_value;
+        // c_array[nb_c_value]=3.67e-3; // 0.186; 3.67e-3; 4.79e-5; 4.24e-7;
+    }
+    else{
+        // default value to not let it empty, it won't be use later.
+        c_array[nb_c_value] = 0.0;
+    }
 
     dealii::ConvergenceTable convergence_table_density;
```

Comment:
These tables are common to the standard h_refinement test so could you move them after line 162?